Marlin/pins_MEGATRONICS.h

```diff
@@ -74,7 +74,8 @@
 #define TEMP_1_PIN         15   // ANALOG NUMBERING
 #define TEMP_BED_PIN       14   // ANALOG NUMBERING
 
-#define BEEPER_PIN         33   // AUX-4
+// AUX-4
+#define BEEPER_PIN         33
```

Comment:
This is better as an end-of-line comment. There it says "this pin number corresponds to AUX-4" whereas placing it above the line says "The following line is your AUX-4."